packages/ui/src/molecules/Incentives/Incentives.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { forwardRef } from 'react'
+import type { FC, ReactNode, AriaAttributes } from 'react'
+
+import type { IncentivesPureProps } from './IncentivesPure'
+import IncentivesPure from './IncentivesPure'
+import List from '../../atoms/List'
+
+interface ListItemProps {
+  testId: string
+  children: ReactNode
+}
+
+const ListItem: FC<ListItemProps> = ({ children, ...otherProps }) => {
+  if (React.isValidElement(children)) {
+    return <li>{React.cloneElement(children, otherProps)}</li>
+  }
+
+  return <li>{children}</li>
+}
+
+export interface IncentivesProps extends IncentivesPureProps {
+  /**
+   * ID to find this component in testing tools (e.g.: cypress,
+   * testing-library, and jest).
+   */
+  testId?: string
+  /**
+   * Defines a string value that labels the current element.
+   */
+  'aria-label'?: AriaAttributes['aria-label']
+}
+
+const Incentives = forwardRef<HTMLDivElement, IncentivesProps>(
+  function Incentives(
+    {
+      testId = 'store-incentives',
+      'aria-label': ariaLabel = 'Incentives',
+      children,
+      ...otherProps
+    },
+    ref
+  ) {
+    return (
+      <IncentivesPure
+        ref={ref}
+        testId={testId}
+        aria-label={ariaLabel}
+        {...otherProps}
+      >
+        <List data-incentives-list role="list" variant="unordered">
```

Comment:
This MDN WAI-ARIA Roles article mentions we [should avoid](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles#:~:text=list%20(use%20either%20%3Cul%3E%20or%20%3Col%3E%20instead)) using `role="list"` when using the semantic HTML equivalent elements `ul`s and `ol`s. Do you think we can remove it?

```suggestion
        <List data-incentives-list variant="unordered">
```

Reply:
@filipewl yes, I just forgot to remove, thanks!